ts/IterableInboxDataModel.ts

```diff
@@ -58,6 +60,27 @@ class IterableInboxDataModel {
         )
     }
 
+    // inbox session tracking functions
+
+    startSession() {
+        RNIterableAPI.startSession(this.getCurrentVisibleRows())
+    }
+
+    endSession() {
+        RNIterableAPI.endSession()
+    }
+
+    updateVisibleRows() {
+        RNIterableAPI.updateVisibleRows(this.getCurrentVisibleRows())
+    }
+
+    // private/internal
+
+    private getCurrentVisibleRows(): InboxImpressionRowInfo[] {
+        console.log("jay getCurrentVisibleRows")
+        return []
+    }
+
```

Comment:
So getCurrentVisibleRows will be a function that is created on a future PR from my understanding.